client/jira.go

```diff
@@ -120,6 +128,38 @@ func CommentIssue(issueId, comment string) (domain.Issue, error) {
 	return issue, nil
 }
 
+func Transition(issueId, transitionId string) (error, string) {
+	path := fmt.Sprintf("/rest/api/2/issue/%s/transitions", issueId)
+	transitionRequest := prepare(transitionId)
```

Comment:
seriously no need for `prepare` just do
`	transReq := transitionRequest{
		Transition: transition{
			Id: transitionId,
		},
	}`

Reply:
done